### DISCUSSION TITLE
pnpm update command

### DISCUSSION THREAD

**Discussion by isitreallyalive** [pnpm/pnpm#4617] [Ideas]
Just for the matter of simplicity, why is there not a command like `pnpm upgrade` or `pnpm up` to update pnpm? Writing out `pnpm add -g pnpm` is verbose.

**Comment by zkochan** [comment]
Because the `pnpm update` and `pnpm upgrade` commands are currently used to update dependencies.

So if we add a command for updating pnpm, it should be named differently.

Yarn had a command called [self-update](https://classic.yarnpkg.com/en/docs/cli/self-update). Maybe we could have a similar command.
